tasks.py

```diff
@@ -551,7 +534,7 @@ def release(ctx, version, skip_release_notes=False):
     is_patch_release = version.patch != 0
 
     # Check that we have release notes for the desired version.
-    run("git checkout main", echo=True)
+    # run("git checkout main", echo=True)
```

Comment:
Not sure why we need to be checkout main branch, but if it's not needed, why don't remove this line?